shared/common-adapters/oriented-image.desktop.js

```diff
@@ -244,14 +244,15 @@ class OrientedImage extends React.Component<Props, State> {
   render() {
     return (
       <React.Fragment>
-        <canvas ref={el => (this._canvasRef = el)} style={styleCanvas} />
         {this.state.srcTransformed && (
           <ImageComponent
             src={this.state.srcTransformed}
             style={this.props.style}
+            onDragStart={this.props.onDragStart}
             onLoad={this.props.onLoad}
           />
         )}
+        <canvas ref={el => (this._canvasRef = el)} style={styleCanvas} />
```

Comment:
what is this change for?

Reply:
I need a reference to the image (not the canvas).

```
_imageSetRef = (ref: ?OrientedImage) => {
  this._image = ref
}
```

That was giving me a canvas before this change.